### ISSUE TITLE
Life insurance rewriting batch 2: Benefits of VGLI

### ISSUE THREAD

**Issue by DanielleThierryUSDSVA** [department-of-veterans-affairs/va.gov-team#8441]
# User story
As a Veteran deciding on a life insurance policy, I need to understand how VGLI compares to commercial life insurance options.

# Details

## Process steps to completion

**Previous steps completed before this was paused in July 2019:**

- [x] Strategy guidance
- [x] First draft complete (not GitHub)

**Steps needed to complete this work now:**

- [x] Confirm original IA/content strategy is still valid (Mikki, Danielle, Laura)
- [x] Do a side-by-side comparison of the rewritten & legacy pages to bring over any new/different info since it's been so long since start of work (Laura)
- [x] Identify and address any potential content anomalies in preparation for moving to Drupal (Laura, Randi)
- [x] Finalize existing GitHub draft (Laura) 
- [x] Migrate to Drupal (Laura)
- [x] Add cross links to this page from other appropriate pages (Laura, Randi)

### IA details
_(note: IA and specifically urls may change based on l4 initiative)_

We're going to bring key information about the benefits of VGLI in comparison to other life insurance onto the existing VA.gov [VGLI page](https://www.va.gov/life-insurance/options-eligibility/vgli/).

### Content details
_(note: may need to adjust structure to work in Drupal components, as these pages were started prior to Drupal migration)_

- **Source page:** https://www.benefits.va.gov/insurance/vgli_rates_compare_vgli.asp
- **SEO keywords:** n/a
- **What's needed:** Add a new question beneath "What life insurance benefits can I get with VLGI?" that reads something to the effect of: "Why might I want to consider VGLI over other commercial insurance options?" and highlight with a bulleted list some of the key benefits. Then, we're going to recommend that we link to a PDF version of the "worksheet" that people can print out to compare. We won't have the PDF yet, so we can leave that link off for now until we confirm with SMEs that they agree with this approach (and create the PDF). 

## Draft copy to work with

H2: Why might I want to consider VGLI over commercial insurance options?

Depending on your life insurance needs, you may want to compare VGLI benefits to those of commercial policy options. When you consider one or more private, commercial life insurance options, you can compare them against the benefits you’d get through VGLI. You can use the worksheet on this page to make direct comparisons.

Here are some of the key benefits of choosing VGLI over many commercial life insurance policies. VGLI gives you:

- The ability to get life insurance approved without any health review regardless of how serious your health conditions might be. To be eligible, however, you’ll need to apply within 240 days after separating from your service. Many commercial plans will ask you for a health review, and if you have certain medical conditions, you may not be approved for the commercial plan. Learn what conditions may make you uninsurable under a private plan.
- Coverage for mental health, PTSD, and traumatic brain injury (TBI).
- The same premium rates for men and women.
- The same premium rates for smokers and nonsmokers.
- Coverage for your entire life, unless you opt out.

You can print out this chart to compare what VGLI offers next to any commercial policy you’re considering. 

Print out VGLI life insurance comparison worksheet.

**Comment by DanielleThierryUSDSVA** [comment]
Adding info from our meeting:

- Check to determine if we need to incorporate any relevant contact information from this page (https://www.benefits.va.gov/INSURANCE/resources-contact.asp) per open question document.

**Comment by lwalsh617** [comment]
Put Drupal preview link and Word doc in ticket 11234. Closing this ticket.

**Comment by lwalsh617** [comment]
Re-opening this ticket. We removed the new Q&A copy (see attached screen shot) from the Drupal page as we needed to publish it and the link to preexisting conditions is another page we need to create. Once we get SME approval and are ready to publish the preexisting conditions page, we can drop this Q&A back in and republish this page. 
![VGLI new QA](https://user-images.githubusercontent.com/53535663/104388955-67880980-5508-11eb-95c2-6183c15a9b9d.png)
